csbmysql/resource_binding_user_test.go

```diff
@@ -241,15 +241,8 @@ func checkSSLCipher(requireSSL bool) resource.TestCheckFunc {
 		Expect(err).NotTo(HaveOccurred())
 		Expect("Ssl_cipher").To(Equal(res.VariableName))
```

Comment:
Not that you changed this, but this reads the wrong way around! It should be:
```go
Expect(res.VariableName).To(Equal("Ssl_cipher"))
```

Reply:
Agreed. Lesson learned.
Same reasoning as the `SatisfyAny` vs `ContainElement` matchers.

Passing the "value known in advance" to the Expect function will usually result in weird and misleading error messages about the intentions of the test.

In a way, I believe the `SatisfyAny` vs `ContainElement` was much worse than this one, mainly because if I understand this test correctly, it seems very unlikely to fail.
Also, it seems to me that this test is not testing `terraform-provider-csbmysql` but the `github.com/go-sql-driver/mysql` implementation. It is just checking that the resulting column `SHOW STATUS LIKE 'Ssl_cipher'` was correctly aliased as "Ssl_cipher".

Anyway, I believe it is worth fixing it, and it probably doesn't hurt to keep it around (even if it's just to ensure we don't accidentally modify the `SHOW STATUS LIKE 'Ssl_cipher'` query).